package-exclude.json

```diff
@@ -28,7 +28,8 @@
         "surge",
         "webpack-bundle-analyzer",
         "tabbable",
-        "@adobe/css-tools"
+        "@adobe/css-tools",
+        "victory-core"
```

Comment:
Why is it excluded?

Reply:
I am not seeing this packaged today so not sure we want to package it..

Reply:
@evgeni Is this supposed to exclude only dev dependencies?

Reply:
I'm unsure about this change, but the list must remain sorted.